Robot_Adapter/Delete/_Delete.cs

```diff
@@ -45,8 +46,13 @@ protected override int Delete(Type type, IEnumerable<object> ids)
                 success = DeleteNodes(ids);
             if (type == typeof(Bar))
                 success = DeleteBars(ids);
-            if (type == typeof(BH.oM.Structure.Loads.Loadcase)) 
+            if (type == typeof(BH.oM.Structure.Loads.Loadcase))
+            {
+                BH.Engine.Reflection.Compute.RecordWarning("Note that when all cases are deleted in Robot, combinations are deleted also");
```

Comment:
Would probably move this warning as well. It currently shows up even if an empty list of ids is provided, which makes it always show up when pushing cases as well as combinations.

![image](https://user-images.githubusercontent.com/22005920/62145865-7965f480-b2f4-11e9-9777-bdb77420c581.png)